site/src/xServices/workspace/workspaceXService.ts

```diff
@@ -170,7 +170,7 @@ export const workspaceMachine = createMachine(
           ],
           onError: [
             {
-              actions: ["assignRefreshTemplateError", "displayRefreshTemplateError"],
+              actions: ["assignRefreshTemplateWarning", "displayRefreshTemplateWarning"],
               target: "error",
```

Comment:
Do you want to go to an `error` state here, if this is just a warning?

Reply:
For now it doesn't matter, because we go to an error state in the Workspace Page anyway if that warning exists. Definitely something to fix when I change up workspace errors.